app/storage/metadata_parser.py

```diff
@@ -88,6 +88,10 @@ def _validate_metadata_is_present(metadata, required_metadata):
     :param required_metadata:
     """
     for metadata_field in required_metadata:
```

Comment:
These technically are no longer `required_metadata`?

Reply:
`metadata` and `required_metadata` get passed through to us, so maybe we should look at doing something with optional/required? Or should we simply just be renaming?

Reply:
`schema_metadata` maybe?

Reply:
i think we need to look at the naming of a lot of the params in this file